mlflow/server/js/src/model-registry/components/SchemaTable.tsx

```diff
@@ -54,26 +129,33 @@ export class SchemaTableImpl extends React.PureComponent<Props> {
     );
   };
 
-  getSchemaTypeRepr = (schemaTypeSpec: any) => {
-    let { type } = schemaTypeSpec;
-    if (schemaTypeSpec.type === MODEL_SCHEMA_TENSOR_TYPE) {
-      type = `Tensor (dtype: ${schemaTypeSpec['tensor-spec'].dtype}, shape: [${schemaTypeSpec['tensor-spec'].shape}])`;
+  getSchemaTypeRepr = (schemaTypeSpec: ColumnSpec | TensorSpec): string => {
+    const { type } = schemaTypeSpec;
+
+    let repr: string = type;
+    if (schemaTypeSpec.type === 'tensor') {
+      repr = `Tensor (dtype: ${schemaTypeSpec['tensor-spec'].dtype}, shape: [${schemaTypeSpec['tensor-spec'].shape}])`;
```

Comment:
Can we `getTensorTypeRepr` here as well?

Reply:
i forgot i was supposed to delete this function actually—it's been changed to `formatColumnSchema`